src/openMVG/multiview/solver_resection_p5pfr.cpp

```diff
@@ -0,0 +1,373 @@
+/*

```

Comment:
The project has a license which applies to the whole project.
AFAIK, these headers are optional. The goal is just to avoid ambiguities and ensure that if someone receive a partial copy, he will know the license.